Feature/route data/get children (#1493)
plentymarkets/terra-components: CHANGELOG.md

> # 6.0.1 Unreleased

Reviewer: ```suggestion
# 6.1.0 Unreleased
```
Ist ja nen Feature also ein Minor-Release

### Feature

> -   **route-children-function** implemented getChild() function which returns the children to a route with it's children

Reviewer 2: ```suggestion
-   **route-children-function** implemented `getChildren()` function which returns the children of a given route wether they are lazy loaded or not.
```

> -   **route-children-function** implemented `getChildren()` function which returns the children of a given route wether they are lazy loaded or not.

Reviewer 3: ```suggestion
-   **route-children-function** implemented `getChildren()` function which returns the children of a given route wether they are lazy loaded or static.
```

> -   **route-children-function** implemented `getChildren()` function which returns the children of a given route wether they are lazy loaded or static.

Reviewer: ```suggestion
-   **route** implemented `getChildren()` function which returns the children of a given route whether they are lazy loaded or static.
```
